CMakeLists.txt

```diff
@@ -23,6 +23,7 @@ IF(MSVC)
 	ADD_DEFINITIONS(-D_HAS_AUTO_PTR_ETC) # for Luabind on C++17
 
 	SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /MP")
+	ADD_DEFINITIONS( "/W0 /D_CRT_SECURE_NO_WARNINGS /wd4005 /wd4996 /nologo /Os")
```

Comment:
For this wouldn't it be better to set this in the done script instead of enforcing it on everyone's build?

Reply:
This hides the sheer volume of warnings that spam Windows that no one ever does anything about and is not doing anyone any good. 

The flags in here are of benefit to everyone really.